railties/lib/rails/application/configuration.rb

```diff
@@ -130,6 +130,10 @@ def load_defaults(target_version)
             action_dispatch.use_cookies_with_metadata = true
           end
 
+          if respond_to?(:action_mailer)
+            action_mailer.delivery_job = ::ActionMailer::MailDeliveryJob
```

Comment:
This is making ActiveJob be loaded on boot. Can we set the name as string and transform in a constant in the `on_load` hook of `action_mailer`?